### PR TITLE
[670] - [BugTask] Display the name of uploaded/renamed/deleted files

### PR DESCRIPTION
**Pull request by ww-jermaine**
## Issue Link
- https://app.asana.com/0/1203011167276287/1203349758582443/f

## Definition of Done
- [x] Display the name of uploaded file
- [x] Display the name of file before renaming and after renaming
- [x] Display the name of deleted file  
- [x] Fixed error toasts when uploading, renaming and deleting files.

## Notes
- None.

## Pre-condition
- Try to upload a file to check if the name of the uploaded file is displayed.
- Try to rename the file to check if the name of file before and after renaming is displayed.
- Try to delete the file to check if the name of file being deleted is displayed. 
- Try to disable server then perform uploading, renaming and deleting to see if error message is displayed.

## Expected Output
- The toast should display the name of uploaded files
- The toast should display the name of file before and after renaming
- The toast should display the name of deleted file

### Success toasts

https://user-images.githubusercontent.com/109291819/201581600-caabfeb3-308f-4968-9e46-9c6b2c311a34.mp4

### Error toasts

https://user-images.githubusercontent.com/109291819/201631360-ec1252ec-2f9b-46d5-9d9d-f15bf99c17fb.mp4

### CODE REVIEW

File: client/src/components/molecules/FileList/DragFileUploadDialog.tsx [hunk 1]
```diff
@@ -29,7 +29,10 @@ const DragFileUploadDialog: FC<Props> = (props): JSX.Element => {
         if (isError) {
           toast.error(`${error.content}`)
         } else {
-          toast.success('File uploaded successfully!')
+          for (let i = 0; i < acceptedFiles.length; i++) {
+            const filename = acceptedFiles[i].name.split('.').slice(0, -1).join('.')
+            toast.success(`${filename} uploaded successfully!`)
+          }
```
Comment (abduljalilpalala) [inline client/src/components/molecules/FileList/DragFileUploadDialog.tsx]: Use es6 forEach instead.

File: client/src/pages/team/[id]/files.tsx [hunk 2]
```diff
@@ -168,7 +178,10 @@ const FileHeader = (props: FileHeaderProps): JSX.Element => {
         if (isError) {
           toast.error(`${error.content}`)
         } else {
-          toast.success('File uploaded successfully!')
+          for (let i = 0; i < fileUploaded.length; i++) {
+            const filename = fileUploaded[i].name.split('.').slice(0, -1).join('.')
+            toast.success(`${filename} uploaded successfully!`)
+          }
```
Comment (abduljalilpalala) [inline client/src/pages/team/[id]/files.tsx]: Same with my previous comment

**Review by abduljalilpalala — CHANGES_REQUESTED** [review]
Please check comments.

**Review by rogeliojohnoliverio — APPROVED** [review]
Approved 🙇

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✔️

**Pull request merged**